cypress/e2e/permit2.test.ts

```diff
@@ -5,212 +5,192 @@ import { getTestSelector } from '../utils'
 
 /** Initiates a swap. */
 function initiateSwap() {
-  // The swap button is re-rendered once enable, so we must wait until the original button is not disabled to re-select the appropriate button.
+  // The swap button is re-rendered once enabled, so we must wait until the original button is not disabled to re-select the appropriate button.
   cy.get('#swap-button').should('not.be.disabled')
   // Completes the swap.
   cy.get('#swap-button').click()
-  cy.get(getTestSelector('confirm-swap-button')).click()
+  cy.contains('Confirm swap').click()
 }
 
 describe('Permit2', () => {
-  // The same tokens & swap-amount combination is used for all permit2 tests.
+  // The same tokens are used for all permit2 tests.
   const INPUT_TOKEN = DAI
   const OUTPUT_TOKEN = USDC_MAINNET
-  const TEST_BALANCE_INCREMENT = 0.01
 
   beforeEach(() => {
     // Sets up a swap between INPUT_TOKEN and OUTPUT_TOKEN.
     cy.visit(`/swap/?inputCurrency=${INPUT_TOKEN.address}&outputCurrency=${OUTPUT_TOKEN.address}`, {
       ethereum: 'hardhat',
     })
-    cy.get('#swap-currency-input .token-amount-input').type(TEST_BALANCE_INCREMENT.toString())
+    cy.get('#swap-currency-input .token-amount-input').type('0.01')
   })
 
   /** Asserts permit2 has a max approval for spend of the input token on-chain. */
   function expectTokenAllowanceForPermit2ToBeMax() {
     // check token approval
-    return cy
-      .hardhat()
+    cy.hardhat()
```

Comment:
does prettier not run on this files?

Reply:
I think this _was_ done by prettier, as a result of the `return` being removed.